test/test-tetris.js

```diff
@@ -548,84 +548,22 @@ describe("generation tetramino, the size tetramino 5x5", function(){
 		})
 	})
 	
-	it("options vertical tetramino x:2 and y:0-4", function(){
+	it("invalid length tetramino is an error", function(){
```

Comment:
Why do you have horizonatal and vertical error for 3-length tetromino, but only vertical for 5-length tetromino?
Please add test for vertical 5-length tetromino